Makefile

```diff
@@ -39,5 +39,8 @@ lint:
 	diff requirements.txt requirements.commit
 	rm requirements.commit
 
+functional-tests-exitfirst:
```

Comment:
When do we use regular `functional-tests`?
If there is no place don't we want to just change `functional-tests` command to `exitfirst` by default?

Reply:
That's my point.
Don't we want to run the `functional-tests-exitfirst` everywhere so we can stop the CI immediately on fail? 
That would be useful if we run the functional tests a lot.

Reply:
Yes. We also want to run the complete tests to be able to map all failures in some cases too.